src/arrays/arrays_with_methods/hard/q14/MainTest.java

```diff
@@ -0,0 +1,82 @@
+package arrays.arrays_with_methods.hard.q14;
+
+import global.BaseTest;
+import global.MethodTest;
+import global.exceptions.InvalidClauseException;
+import global.tools.CustomAssertions;
+import global.tools.TestOption;
+import global.variables.Clause;
+import global.variables.clauses.NewLine;
+import global.variables.clauses.PlaceHolder;
+import global.variables.clauses.StringLiteral;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+import static global.utils.ArrayUtil.arrayToInput;
+
+public class MainTest extends BaseTest {
+    // Java
+
+    public static final double a = 2;
+    public static final double r = 0.5;
+
+    public Clause[] testSentence() {
+        TestOption.isInputTest = true;
+        TestOption.defaultInput = "10";
+        return new Clause[]{
+                new StringLiteral("Enter a value for n: "),
```

Comment:
Because this is a java question, instead of just having the space at the end of the string, we use the Optional wrapper on another StringLiteral with the space so that the student's code does not fail the test if they forget this.